tests/test-rest-posts-controller.php

```diff
@@ -588,6 +588,67 @@ public function test_get_items_valid_date() {
 		$this->assertEquals( $post2, $data[0]['id'] );
 	}
 
+	public function test_get_items_valid_tax_query_in_filter() {
+		$term1 = $this->factory->term->create( array( 'taxonomy' => 'category', 'name' => 'basie' ) );
+		$post1 = $this->factory->post->create();
```

Comment:
Not a blocker, but no need to assign the `$post1` and `$post3` variables since you don't use them.